src/main/webapp/createYourOwn.html

```diff
@@ -22,6 +22,51 @@
             </nav>
         </div>
       </header>
+      <h1>New Initiative</h1>
+      <hr style="height:4px;background-color:#F8944C;border-width:0px;color:#F8944C;width:300px;margin-bottom:30px">
+      <div id="new-initiative">
+          <form id="newInitiativeForm" action="myUploadUrl" method="POST" enctype="multipart/form-data">
+            <p class="initiative-input">Title:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-title">
+            <br/><br/>
+            <p class="initiative-input">Insert Initiative Image:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="file" name="initiative-image">
+            <br/><br/>
+            <p class="initiative-input">Location:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-location">
+            <br/><br/>
+            <p class="initiative-input">Start Date:
+                <img src="images/asterisk.png" style="width:10px;height:10px;"></p>
+            <input type="text" name="initiative-start-date" value="MM/DD/YYYY">
```

Comment:
I believe there is a type option called date for input tag. If you declare the type to be date, internally, html will ensure a certain format they expect for a date.